py_mmd_tools/mmd_to_nc.py

```diff
@@ -0,0 +1,256 @@
+        except IndexError:
+            out['id'] = value
+
+        return out
+
+    def update_nc(self):
+        """
+        Update a netcdf file global attributes.
+        """
+
+        acdd = {}
+
+        # Loop on expected MMD elements
+        # ie from mmd_elements.yaml = kind-off mapping between acdd and mmd
+        for mmd_element in self.mmd_yaml:
+
+            elements = self.tree.findall('mmd:' + mmd_element, self.namespaces)
+
+            if len(elements) == 0:
+                print(f'{mmd_element} not found in input MMD file')
+                continue
+
+            # Loop on  MMD elements found
+            for elem in elements:
+
+                tag = ET.QName(elem).localname
+
+                if mmd_element in ['title', 'abstract']:
+                    match = self.get_title_abstract(elem, tag)
+                    acdd = self.update_acdd(acdd, match)
+
+                elif mmd_element == 'keywords':
+                    match, sep = self.get_keyword(elem)
+                    acdd = self.update_acdd(acdd, match, sep)
+
+                elif mmd_element == 'last_metadata_update':
+                    match, sep = self.get_last_metadata_update(elem)
+                    acdd = self.update_acdd(acdd, match, sep)
+
+                elif mmd_element == 'metadata_identifier':
+                    match = self.get_metadata_identifier(elem)
+                    acdd = self.update_acdd(acdd, match)
+
+                elif mmd_element == 'personnel':
+                    match, sep = self.get_personnel(elem)
+                    acdd = self.update_acdd(acdd, match, sep)
+
+                # dataset_citation = repeat of mmd fields already processed elsewhere
+                elif mmd_element == 'dataset_citation':
+                    continue
+
+                # No subselements
+                elif len(list(elem)) == 0:
+                    match, sep = self.process_elem(elem, self.mmd_yaml, tag)
+                    acdd = self.update_acdd(acdd, match, sep)
+
+                # Subselements processed independently
+                else:
+
+                    for subelem in list(elem):
+
+                        subtag = ET.QName(subelem).localname
+
+                        if len(list(subelem)) > 0:
+                            for subsubelem in list(subelem):
+                                subsubtag = ET.QName(subsubelem).localname
+                                match, sep = self.process_elem(subsubelem,
+                                                               self.mmd_yaml[tag][subtag],
+                                                               subsubtag)
+                                acdd = self.update_acdd(acdd, match, sep)
+                        else:
+                            match, sep = self.process_elem(subelem, self.mmd_yaml[tag], subtag)
+                            acdd = self.update_acdd(acdd, match, sep)
+
+        # Conventions
```

Comment:
I'm not really sure it's the time and place to add the convention element.
What do you think @mortenwh

Reply:
Yes  - you can use it. Just define the ACDD version in your code, maybe as a variable.